examples/plot_vector_testing.py

```diff
@@ -0,0 +1,205 @@
+"""
+Testing Vector Plots Created Using Geopandas with MatPlotCheck
+==============================================================
+
+This vignette will show you how to use Matplotcheck to test spatial vector data
+plots created using Geopandas.
+
+"""
+
+################################################################################
+# Setup
+# -----
+# To begin, import the libraries that you need.
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from shapely.geometry import Polygon, LineString
+import geopandas as gpd
+from matplotcheck.vector import VectorTester
+
+################################################################################
+# Create Geometry Objects
+# -----------------------
+# To run this test, you first must plot some example data. Below are GeoPandas
+# dataframes created to replicate data that may be used in an earth data
+# science exercise. The polygons could be study areas, the lines could be roads
```

Comment:
Minor changes: "... in an earth or environmental data science exercise. The polygons could be study areas, while the lines could be roads and streams near those study areas. The points could be measurements within the study areas."

Note the correct spelling of measurements.